examples/custom_mouse_functions.py

```diff
@@ -50,16 +51,15 @@ def get_connected_component_shape(layer, event):
             msg = f'clicked at {cords} on background which is ignored'
         print(msg)
 
-
     # Handle click or drag events separately
     @labels_layer.mouse_drag_callbacks.append
-    def click_drag(layer, event):
+    def click_drag(layer, cursor_event):
         print('mouse down')
         dragged = False
         yield
         # on move
-        while event.type == 'mouse_move':
-            print(event.pos)
+        while cursor_event.type == 'mouse_move':
```

Comment:
change all of those `== 'mouse_mouve'` to `== CursorEventType. MOUSE_MOUVE` ? maybe ?